alot/commands/__init__.py

```diff
@@ -22,6 +22,20 @@ def __init__(self):
         self.undoable = False
         self.help = self.__doc__
 
+        mode, cmdname = reverse_lookup_command(self.__class__)
```

Comment:
please use `type(self)` instead of accessing the dunder directly. it would probably be best to just store the result for both class and the name. This aplies to the rest of the patch.